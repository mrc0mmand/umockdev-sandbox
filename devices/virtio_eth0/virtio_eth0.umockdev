P: /devices/pci0000:00/0000:00:05.0/virtio2/net/eth0
E: INTERFACE=eth0
E: IFINDEX=2
E: SUBSYSTEM=net
E: ID_NET_NAMING_SCHEME=v249
E: ID_NET_NAME_MAC=enx5254007251d9
E: ID_NET_NAME_PATH=enp0s5
E: ID_NET_NAME_SLOT=ens5
E: ID_BUS=pci
E: ID_VENDOR_ID=0x1af4
E: ID_MODEL_ID=0x1000
E: ID_PCI_CLASS_FROM_DATABASE=Network controller
E: ID_PCI_SUBCLASS_FROM_DATABASE=Ethernet controller
E: ID_VENDOR_FROM_DATABASE=Red Hat, Inc.
E: ID_MODEL_FROM_DATABASE=Virtio network device
E: ID_PATH=pci-0000:00:05.0
E: ID_PATH_TAG=pci-0000_00_05_0
E: ID_NET_DRIVER=virtio_net
E: ID_NET_LINK_FILE=/usr/lib/systemd/network/99-default.link
E: ID_NET_NAME=eth0
E: SYSTEMD_ALIAS=/sys/subsystem/net/devices/eth0
E: TAGS=:systemd:
E: CURRENT_TAGS=:systemd:
A: addr_assign_type=0\n
A: addr_len=6\n
A: address=52:54:00:72:51:d9\n
A: broadcast=ff:ff:ff:ff:ff:ff\n
A: carrier=1\n
A: carrier_changes=2\n
A: carrier_down_count=1\n
A: carrier_up_count=1\n
A: dev_id=0x0\n
A: dev_port=0\n
L: device=../../../virtio2
A: dormant=0\n
A: duplex=unknown\n
A: flags=0x1003\n
A: gro_flush_timeout=0\n
A: ifalias=
A: ifindex=2\n
A: iflink=2\n
A: link_mode=0\n
A: mtu=1500\n
A: napi_defer_hard_irqs=0\n
A: netdev_group=0\n
A: operstate=up\n
A: power/control=auto\n
A: power/runtime_active_time=0\n
A: power/runtime_status=unsupported\n
A: power/runtime_suspended_time=0\n
A: proto_down=0\n
A: speed=-1\n
A: statistics/collisions=0\n
A: statistics/multicast=0\n
A: statistics/rx_bytes=329289\n
A: statistics/rx_compressed=0\n
A: statistics/rx_crc_errors=0\n
A: statistics/rx_dropped=32\n
A: statistics/rx_errors=0\n
A: statistics/rx_fifo_errors=0\n
A: statistics/rx_frame_errors=0\n
A: statistics/rx_length_errors=0\n
A: statistics/rx_missed_errors=0\n
A: statistics/rx_nohandler=0\n
A: statistics/rx_over_errors=0\n
A: statistics/rx_packets=936\n
A: statistics/tx_aborted_errors=0\n
A: statistics/tx_bytes=96293\n
A: statistics/tx_carrier_errors=0\n
A: statistics/tx_compressed=0\n
A: statistics/tx_dropped=0\n
A: statistics/tx_errors=0\n
A: statistics/tx_fifo_errors=0\n
A: statistics/tx_heartbeat_errors=0\n
A: statistics/tx_packets=781\n
A: statistics/tx_window_errors=0\n
A: testing=0\n
A: threaded=0\n
A: tx_queue_len=1000\n
A: type=1\n

P: /devices/pci0000:00/0000:00:05.0/virtio2
E: DRIVER=virtio_net
E: MODALIAS=virtio:d00000001v00001AF4
E: SUBSYSTEM=virtio
A: device=0x0001\n
L: driver=../../../../bus/virtio/drivers/virtio_net
A: features=1110010111111111111101010000110010000000000000000000000000000000\n
A: modalias=virtio:d00000001v00001AF4\n
A: power/control=auto\n
A: power/runtime_active_time=0\n
A: power/runtime_status=unsupported\n
A: power/runtime_suspended_time=0\n
A: status=0x0000000f\n
A: vendor=0x1af4\n

P: /devices/pci0000:00/0000:00:05.0
E: DRIVER=virtio-pci
E: PCI_CLASS=20000
E: PCI_ID=1AF4:1000
E: PCI_SUBSYS_ID=1AF4:0001
E: PCI_SLOT_NAME=0000:00:05.0
E: MODALIAS=pci:v00001AF4d00001000sv00001AF4sd00000001bc02sc00i00
E: SUBSYSTEM=pci
E: ID_PCI_CLASS_FROM_DATABASE=Network controller
E: ID_PCI_SUBCLASS_FROM_DATABASE=Ethernet controller
E: ID_VENDOR_FROM_DATABASE=Red Hat, Inc.
E: ID_MODEL_FROM_DATABASE=Virtio network device
A: ari_enabled=0\n
A: broken_parity_status=0\n
A: class=0x020000\n
H: config=F41A0010070510000000000200000000E1C000000020BDFE00000000000000000C8000FE0000000000000000F41A01000000B8FE98000000000000000A010000090010010400000000000000001000000940100304000000001000000010000009501004040000000020000000100000096014020400000000300000001000000400000009701405000000000000000000000000000000001184028001000000010800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
A: consistent_dma_mask_bits=64\n
A: d3cold_allowed=0\n
A: device=0x1000\n
A: dma_mask_bits=64\n
L: driver=../../../bus/pci/drivers/virtio-pci
A: driver_override=(null)\n
A: enable=1\n
L: firmware_node=../../LNXSYSTM:00/LNXSYBUS:00/PNP0A03:00/device:05
A: irq=10\n
A: local_cpulist=0-3\n
A: local_cpus=f\n
A: modalias=pci:v00001AF4d00001000sv00001AF4sd00000001bc02sc00i00\n
A: msi_bus=1\n
A: msi_irqs/29=msix\n
A: msi_irqs/30=msix\n
A: msi_irqs/31=msix\n
A: numa_node=-1\n
A: power/control=on\n
A: power/runtime_active_time=62741\n
A: power/runtime_status=active\n
A: power/runtime_suspended_time=0\n
A: power_state=D0\n
A: resource=0x000000000000c0e0 0x000000000000c0ff 0x0000000000040101\n0x00000000febd2000 0x00000000febd2fff 0x0000000000040200\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x00000000fe008000 0x00000000fe00bfff 0x000000000014220c\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x00000000feb80000 0x00000000febbffff 0x0000000000046200\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n0x0000000000000000 0x0000000000000000 0x0000000000000000\n
A: revision=0x00\n
A: subsystem_device=0x0001\n
A: subsystem_vendor=0x1af4\n
A: vendor=0x1af4\n

