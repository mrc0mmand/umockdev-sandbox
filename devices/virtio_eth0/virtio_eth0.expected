ID_NET_NAME_MAC=enx5254007251d9
ID_NET_NAME_PATH=enp0s5
ID_NET_NAME_SLOT=ens5
